### DISCUSSION TITLE
能不能编译一个linux arm版出来

### DISCUSSION THREAD

**Discussion by Naoki19880107** [getlantern/lantern-forum-cn#271] [Q&A]
能不能编译一个linux arm版出来

**Comment by MaxMKMax** [comment]
@Naoki19880107 我可以帮您反馈一下这个需求. 但前提是您要提交一个应用内报告关于您的需求, 然后给我您的票号.

**Reply by Naoki19880107** [reply]
70752656

**Reply by MaxMKMax** [reply]
@Naoki19880107 
您好. 我已经帮您提交您的要求上去了. 等科技部门看怎么计划弄这个了. 敬请期待!

**Comment by xu-jan** [comment]
请问arm架构的安装包什么时候能出来呀

**Reply by MaxMKMax** [reply]
@janhsuxu 您好. 我帮您问问看技术团队再给您答复.

**Reply by xu-jan** [reply]
请问怎么样啦，有这个计划排期吗
